packages/contentful--app-scripts/utils/analytics.js

```diff
@@ -0,0 +1,28 @@
+const Analytics = require('analytics-node');
+
+// Public write key scoped to data source
+const SEGMENT_WRITE_KEY = 'define me';
+
+/**
+ *
+ * @param {object} properties tracking properties
+ * @param {string} properties.command triggered command e.g create-app-definition, upload, etc.
+ * @param {boolean} properties.ci value if --ci flag has been set
+ * @returns
+ */
+function track(properties) {
+  if (process.env.DISABLE_ANALYTICS) {
+    return;
+  }
+
+  const client = new Analytics(SEGMENT_WRITE_KEY);
```

Comment:
Why do we want to create new client every time we execute `track`?

Reply:
With this being a CLI script, it will anyway create a new client for every run, even if we wrap it in another class and instantiate on every call.